### ISSUE TITLE
Question: Documentation - Using this vs just enabling Mac deployment in a React Native project?

### ISSUE THREAD

**Issue by kyle-ssg** [microsoft/react-native-macos#622]
As it stands, I can init a react native project @0.62.0, enable both iPad and Mac support and run the application without interacting or following any of the guides from react-native-macos and it runs successfully.

The fact that I can do this with seemingly no issues brings up a few key questions that could maybe do with being communicated.

- I understand that this is a fork of React Native, does this periodically get merged into React Native?
- What is the benefit of using this and also having a separate XCode project for Mac if running within a vanilla React Native project already works?

**Edit:**
I could be way off here but I think I understand. Is it that this is purely Mac OS and doesn't use Catalyst vs React Native which could? Hence this wouldn't support libraries that use UIKit and is the reason why we can now use vanilla React Native?

If that is the case, I think it's definitely noteworthy to React Native developers looking to create a Mac OS app and could also be the defacto suggestion of people raising bugs relating to libraries that use incompatible frameworks.

**Comment by alloy** [comment]
Hey, thanks for the question!

> I could be way off here but I think I understand. Is it that this is purely Mac OS and doesn't use Catalyst vs React Native which could?

Correct, this is an ‘AppKit’ based implementation of React Native for macOS, not a ‘UIKit’ based one such as Catalyst allows you to make.

> What is the benefit of using this and also having a separate XCode project for Mac if running within a vanilla React Native project already works?

Much has been written about the differences between Catalyst or full AppKit based apps in general, i.e. this isn’t really related to the usage of [any flavour of] React Native, so it’s probably best to research if those differences make sense for your requirements.

A contrived summary is that Catalyst is in principle meant to easily make iOS apps available on macOS, and only provides a few more APIs allowing _some_ access to desktop specific features.

> Hence this wouldn't support libraries that use UIKit and is the reason why we can now use vanilla React Native?

I’m unsure what you mean, could you rephrase?

**Comment by kyle-ssg** [comment]
> I’m unsure what you mean, could you rephrase?

Sorry yeah I was a bit unclear here I just meant that this library wouldn't allow you to use anything that references UI Kit but generally using a RN/Catalyst approach would.

Seems fine to close this though maybe as I say, highlighting the difference somewhere may be good for developers unfamiliar with AppKit v Catalyst

**Comment by alloy** [comment]
> maybe as I say, highlighting the difference somewhere may be good for developers unfamiliar with AppKit v Catalyst

Thanks for the feedback! Much appreciated 🙏 

cc @harinikmsft @kikisaints Any thoughts on how/where to best communicate this on the site?

**Comment by tom-un** [comment]
> > maybe as I say, highlighting the difference somewhere may be good for developers unfamiliar with AppKit v Catalyst
> 
> Thanks for the feedback! Much appreciated 🙏
> 
> cc @harinikmsft @kikisaints Any thoughts on how/where to best communicate this on the site?

I was thinking of adding language to the wiki, and perhaps a link from the readme.me to the wiki.   This question has come up a couple times before.   The biggest reason for an AppKit based react-native-macos versus a UIKit Catalyst react-native, is that Catalyst is an "all or nothing" UIKit solution, meaning that if you have a brownfield AppKit based Mac app its not possible to add a Catalyst module.
